xxd: add page (#4485)
tldr-pages/tldr: pages/linux/xxd.md

# xxd

> Make a hexdump or do the reverse.
> More information: <https://packages.debian.org/buster/xxd>.

> - Display Hex representation of a file:

Reviewer: Perhaps we shall adjust this to make the casing concise.
```suggestion
- Display hex representation of a file:
```

Author: I agree it certainly should consistent overall!

> `xxd {{file}}`

Reviewer: Does this expect only the filename or a path?

Author: A path to the file to be specific.
If the file is in current working dir then just file name is enough.

> `xxd {{file}}`

Reviewer 2: ```suggestion
`xxd {{path/to/file}}`
```

> - Convert Hexdump into binary:

Reviewer: same as above
```suggestion
- Convert hex dump into binary:
```

Author: that's neat!

> `xxd -r {{file}}`

Reviewer 2: ```suggestion
`xxd -r {{path/to/file}}`
```

> - Produce Binary dump:

Reviewer: same as above
```suggestion
- Produce a binary dump:
```

`xxd -b {{file}}`

- Print everything but the first three lines (hex  0x30  bytes):

> `xxd -s 0x30 {{file}}`

Reviewer 2: ```suggestion
`xxd -s 0x30 {{path/to/file}}`
```

> - Print 120 bytes as continuous hexdump with 20 octets per line:

Reviewer: I'm suggesting that because a quick search on "hexdump" would only return results with it separated as ["Hex Dump"](https://pt.wikipedia.org/wiki/Hex_dump).
```suggestion
- Print 120 bytes as continuous hex dump with 20 octets per line:
```

Author: How about we do:
- Print 120 bytes as continuous Hex dump with 20 octets per line:
instead of:
- Print 120 bytes as continuous hex dump with 20 octets per line:
Thanks for the suggestions :)

> `xxd -l 120 -ps -c 20 {{file}}`

Reviewer 2: ```suggestion
`xxd -l {{120}} -ps -c {{20}} {{path/to/file}}`
```
